llvm/test/CodeGen/AMDGPU/unreachable-after-cs-chain.ll

```diff
@@ -0,0 +1,38 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -mtriple=amdgcn--amdpal -mcpu=gfx1200 -verify -amdgpu-unify-divergent-exit-nodes -verify -S %s -o - | FileCheck %s
```

Comment:
Use `llvm-as` for verifier test. Also, this test needs to be moved to where IR verifier tests are.

Reply:
This runs a pass, so I don't think `llvm-as` etc is appropriate.

Reply:
An IR verifier test should not run a pass